zcbor/zcbor.py

```diff
@@ -2588,9 +2588,9 @@ def unique_types(self, mode):
                 else:
                     assert (
                         ''.join(type_names[type_name]) == ''.join(type_def[0])), \
-                        "Two elements share the type name %s, but their implementations are not " \
-                        + "identical. Please change one or both names. One of them is %s" \
-                        % (type_name, pformat(mtype.type_def()))
+                        ("Two elements share the type name %s, but their implementations are not "
```

Comment:
Could we start using format strings now? Boy scout rule style, whenever we change a string we introduce `f"something"` style formatting?

Reply:
Deal :)